package.json

```diff
@@ -24,6 +24,7 @@
         "notify": true
     },
     "devDependencies": {
+        "coveralls": "^3.0.0",
```

Comment:
Remove the caret. We want to ensure it's the same dependency that's always installed so we don't get breaking changes because of any of the dev upgrades.